app/models/notification_blueprint.rb

```diff
@@ -18,6 +18,14 @@ class NotificationBlueprint < ActiveRecord::Base
   validates :expires_in, :numericality => {:greater_than => 0}
   before_validation :set_default_expiry
 
+  def mass_update_expiry
```

Comment:
its used in the discovery notification pr - see https://github.com/theforeman/foreman_discovery/issues/328